server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from .shared_case import SharedTestCase
+from events.models import Event
+
+class EventTest(SharedTestCase):
+    """ Test module for Event model """
+
+    def setUp(self):
+        # event1
+        Event.objects.create(
```

Comment:
Setting the created `Event` to a variable will help you reduce code in future testcases, and reduce the number of database queries you make.
`self.event1 = Event.objects.create(...)`

Reply:
Fixed